api/payara-bom/pom.xml

```diff
@@ -546,7 +546,7 @@
             <dependency>
                 <groupId>io.opentelemetry</groupId>
                 <artifactId>opentelemetry-opentracing-shim</artifactId>
-                <version>${opentelemetry.alpha.version}</version>
+                <version>${opentelemetry.version}</version>
```

Comment:
Looking at the versions available on Maven Central, I guess they promoted this from alpha status